src/ibattle.h

```diff
@@ -20,6 +20,7 @@ lsl/battle/ibattle.h
 #include "user.h"
 #include "userlist.h"
 #include <lslunitsync/optionswrapper.h>
+#include <wx/colour.h>
```

Comment:
where should I put the include and the function exactly. That colour is only used on the battle list, and it's "calculated" only once when it's set with a specific game.

Reply:
ideally in a .cpp file

Reply:
apparently to define the attribute on the .h it needs to know what a wxColour is so it needs the include, and that's needed to avoid having to calculate it every time the row attributes are applied.

other .h files also include wx stuff, for example, battle.h includes <wx/event.h>

Reply:
there, i've removed the include from ibattle.h